### DISCUSSION TITLE
How to select Reviews to confirm all at once?

### DISCUSSION THREAD

**Discussion by Sanookmakmak** [photoprism/photoprism#2630] [Q&A]
The indexer found a larger amount of pictures to confirm and i want to confirm them all together at once but can't find an option for it?

**Comment by lastzero** [comment]
Multi select or disable review in Settings.

**Comment by Sanookmakmak** [comment]
Disabling  Review in settings is a good idea, thank you!

But i did not found out yet how to select all hooks together at once to confirm:

![Snipaste_2022-08-22_12-38-37](https://user-images.githubusercontent.com/24833757/185903575-2303bf14-4803-474f-9050-ea27bd29caf9.png)

**Reply by lastzero** [reply]
Shift or long click. There is no select all button, see existing discussions.

**Comment by Sanookmakmak** [comment]
When i click on the hook at step 4 to confirm the selected pictures, only one picture will be confirmed and the other selected ones are remaining?

![Snipaste_2022-08-22_15-39-35](https://user-images.githubusercontent.com/24833757/185935042-0896e5ed-ed8e-495c-aefd-cd883b299c31.jpg)

**Reply by lastzero** [reply]
Use the context menu (bottom right).

**Reply by lastzero** [reply]
![Screenshot_20220822_161739](https://user-images.githubusercontent.com/301686/185943588-316240d4-d365-4cb1-a8df-5e648111ae12.jpg)
